app/build.gradle

```diff
@@ -5,12 +5,12 @@ android {
     compileSdkVersion 30
     buildToolsVersion "30.0.3"
     defaultConfig {
-        applicationId "com.hazem.wazkar"
+        applicationId "com.hamza.wazkar"
         minSdkVersion 14
         //noinspection OldTargetApi,ExpiredTargetSdkVersion
         targetSdkVersion 30
-        versionCode 12
-        versionName "10.2"
+        versionCode 13
+        versionName "8.2"
```

Comment:
should be 1 or at most 2 not 8 at all

Reply:
don't understand you 
It's just something for google play store

Reply:
you should reset your versioning of app 
if it X.Y.Z
so changing X for major changes 
Y for minor changes 
Z for bug fixes